src/cli_driver.rs

```diff
@@ -363,7 +363,7 @@ struct ProcessingSession {
     /// This is the virtual "CWD" that our filesystem accesses use. It is the
     /// dirname of `primary_input_path`, or an empty path (i.e., corresponding
     /// to the CWD if `primary_input_path` is None.
-    fs_root: PathBuf,
+    #[allow(unused)] fs_root: PathBuf,
```

Comment:
Oh, I see, I misunderstood, I thought the `fs_root` below is this one. I think that it is better to remove it. It's easy to add it back if needed.